bio2zarr/vcf2zarr/icf.py

```diff
@@ -1081,8 +1132,24 @@ def process_partition(self, partition_index):
                     if has_gt:
                         tcw.append("FORMAT/GT", variant.genotype.array())
                     for field in format_fields:
+                        if field.full_name == "FORMAT/LAA":
+                            laa_val = compute_laa_field(variant)
+                            # Convert laa_val to a NumPy array
+                            max_laa_len = max(len(laa) for laa in laa_val)
+                            # At minimum, we want to have at least one value per sample
+                            # so that the field is present.
+                            max_laa_len = max(1, max_laa_len)
+                            laa_val = [
+                                sorted(laa)
+                                + [constants.INT_FILL] * (max_laa_len - len(laa))
+                                for laa in laa_val
+                            ]
+                            laa_val = np.array(laa_val)
+                            tcw.append("FORMAT/LAA", laa_val)
+                            continue
```

Comment:
Minor style thing here, but I'd rather use an explicit ``else`` on the other case than a ``continue`` here. I tend to use these sparingly, and for "early exit" close to the top of the loop